code/backend/pages/03_Delete_Data.py

```diff
@@ -65,6 +66,9 @@
                 files_to_delete = search_handler.delete_files(
                     selected_files,
                 )
+                # Delete from Storage
+                blob_client = AzureBlobStorageClient()
+                blob_client.delete_files(selected_files)
```

Comment:
Now that @liammoat is handling the blob delete event to delete the document from the search index under https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/pull/893, do you think we should just do the blob delete from here, and let the function trigger handle the index delete?

Reply:
Yes will close this PR for now and wait for @liammoat changes to go in